### DISCUSSION TITLE
Function to quickly add a word to custom dictionary

### DISCUSSION THREAD

**Discussion by asymmetric** [minad/jinx#142] [Q&A] [answered]
Thanks for the package, it works like a charm :sparkles: 

It would be very handy if jinx had a function similar to spell-fu's [`spell-fu-word-add`](https://github.com/emacsmirror/spell-fu/blob/e4031935803c66eca2f076dce72b0a6a770d026c/spell-fu.el#L1148), so I wouldn't have to even summon the prompt presented by `jinx-correct` to then type `@`.

**Comment by minad** [comment, marked as answer]
Did you try to use keyboard macros for this - `M-$ @ RET`?
